rpc/src/v1/impls/eth.rs

```diff
@@ -146,6 +146,33 @@ enum PendingTransactionId {
 	Location(PendingOrBlock, usize)
 }
 
+pub fn base_logs<C, M, T: StateInfo + 'static> (client: &C, miner: &M, filter: Filter) -> BoxFuture<Vec<Log>> where
+	C: miner::BlockChainClient + BlockChainClient + StateClient<State=T> + Call<State=T>,
+	M: MinerService<State=T> {
```

Comment:
Sorry for nitpicking but would prefer formatting the where thingy little differently.

```rust
pub fn base_logs<C, M, T: StateInfo + 'static>(client: &C, miner: &M, filter: Filter)
where
    C: miner::BlockChainClient + BlockChainClient + StateClient<State = T> + Call<State = T>,
    M: MinerService<State = T>,
{}
```